### PR TITLE
Implement `getMetadata` for some more Passive scripts

### PR DESCRIPTION
**Pull request by ricekot**
Update the following scripts to implement the `getMetadata()` function:
- passive/find base64 strings.js
- passive/Find Credit Cards.js
- passive/Find Emails.js
- passive/Find Hashes.js
- passive/Find HTML Comments.js

Part of #440.

### CODE REVIEW

**Comment by ricekot** [comment]
Added `codeLink` and `helpLink` to all passive scripts, which were also missed in #446.

File: passive/Find Credit Cards.js [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-    "",
-  ];
-  var cweId = [0, 1];
-  var wascId = [0, 1];
+var ScanRuleMetadata = Java.type(
+  "org.zaproxy.addon.commonlib.scanrules.ScanRuleMetadata"
+);
 
-  // lets make some regular expressions for well known credit cards
-  // regex must appear within /( and )/g
```
Comment (kingthorin) [inline passive/Find Credit Cards.js:19]: Should we keep this and just move it down to L27?

**Review by kingthorin — COMMENTED** [review]
LGTM overall

File: passive/Find Credit Cards.js (hunk 1 above)
Comment (ricekot) [reply passive/Find Credit Cards.js]: Tweaked

**Review by kingthorin — APPROVED** [review]

File: passive/find base64 strings.js [hunk 2]
```diff
@@ -38,40 +50,23 @@ function scan(ps, msg, src) {
...
-          foundstrings.toString(),
-          alertSolution,
-          "",
-          cweId,
-          wascId,
-          msg
-        );
+        helper
+          .newAlert()
+          .setParam("fakeparam" + counter)
```
Comment (thc202) [inline passive/find base64 strings.js:65]: Remove? (It didn't have it before.)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**